torchvision/datasets/_optical_flow.py

```diff
@@ -118,7 +118,7 @@ class Sintel(FlowDataset):
             return a built-in valid mask, such as :class:`~torchvision.datasets.KittiFlow`.
     """
 
-    def __init__(self, root, split="train", pass_name="clean", transforms=None):
+    def __init__(self, root, split="train", pass_name="clean", transforms=None) -> None:
```

Comment:
Well let's type this correctly?

```suggestion
    def __init__(self, root: str, split: str = "train", pass_name: str ="clean", transforms: Optional[Callable] = None) -> None:
```